integreat_cms/cms/views/regions/region_actions.py

```diff
@@ -37,6 +38,51 @@ def delete_region(request, *args, **kwargs):
     """
 
     region = get_object_or_404(Region, slug=kwargs.get("slug"))
+
+    # Check whether region can be safely deleted
+    mirrored_pages = Page.objects.filter(
+        region=region, mirroring_pages__isnull=False
+    ).distinct()
+    if len(mirrored_pages) > 0:
+        messages.error(
+            request,
+            format_html(
+                "{}<ul class='list-disc pl-4'>{}</ul>",
+                _(
+                    "Region could not be deleted, because the following pages are mirrored in other regions:"
+                ),
+                format_html_join(
+                    "\n",
+                    "<li><a href='{}' class='underline hover:no-underline'>{}</a> ({} {})</li>",
+                    [
+                        (
+                            page.best_translation.backend_edit_link,
+                            page.best_translation.title,
+                            _("is mirrored here:"),
+                            format_html_join(
+                                ", ",
+                                "<a href='{}' class='underline hover:no-underline'>{}</a>",
+                                [
+                                    (
+                                        mirrored_page.best_translation.backend_edit_link,
+                                        mirrored_page.best_translation.title,
+                                    )
+                                    for mirrored_page in page.mirroring_pages.all()
+                                ],
```

Comment:
I would suggest to link all pages that mirror the listed page so the service team can directly remove any mirrored relationships if needed.
Just listing the mirrored pages is not very helpful if we don't know where they are embedded from.